src/convertToObject.js

```diff
@@ -13,6 +13,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const strToArr = sourceString.split(';').filter(item => item.length > 4);
+
+  const formattedArr = strToArr
+    .map(element => element.split(':')
+      .map(strItem => strItem.trim()));
```

Comment:
Don't use shortened words in var names.

```suggestion
  const formattedArray = stringToArray
    .map(element => element
      .split(':')
      .map(strItem => strItem.trim());
    );
```